minefana-common/src/main/java/com/github/games647/minefana/common/collectors/PlayerCollector.java

```diff
@@ -73,16 +80,20 @@ public void run() {
     protected abstract int getMaxPlayers();
 
     public void onPlayerJoin(P player) {
-        if (isNew(player)) {
-            newPlayers++;
-        }
-
-        UUID uuid = getUUID(player);
-
-        String locale = getLocale(player);
-        InetAddress address = getAddress(player);
-        ProtocolVersion protocol = getProtocol(player);
-        players.put(uuid, newAnalyticsPlayer(player, locale, address, protocol));
+        ScheduledFuture<?> runLater = scheduler.schedule(new Runnable() {
```

Comment:
This won't work, because BungeeCord doesn't like any external threads that doesn't use the Bungee scheduler. Please make platform dependent method so it uses the specific scheduler.

Reply:
ok will do that